network-map-viewer/src/components/utils/equipmentInfosHandler.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { useCallback } from 'react';
+
+export const useNameOrId = (useName) => {
+    const getNameOrId = useCallback(
+        (infos) => {
+            if (infos != null) {
+                const name = infos.name;
+                return useName && name != null && name.trim() !== ''
+                    ? name
+                    : infos?.id;
+            }
+            return null;
```

Comment:
you are mixing null and undefined 
1 -  if infos is undefined you also should return null (or undefined)  => if (!infos){...}
2 - why testing infos?.id but not infos.name ?